cpu/ezr32wg/Makefile.include

```diff
@@ -1,3 +1,7 @@
 export CPU_ARCH = cortex-m4f
 
+USEMODULE += pm_cortexm_common_set_lowest
+USEMODULE += pm_cortexm_common_off
```

Comment:
don't all cortexm  boards use the cortexm_common off and reboot?

Reply:
I think so